openapi/components/schemas/RuleActionOfferPurchaseBump.yaml

```
@@ -1,6 +1,6 @@
 description: Offer Purchase Bump.
```

nitpick: capitilization
```suggestion
description: Offer purchase bump.
```